src/table/hooks/__tests__/use-column-widths.spec.ts

```diff
@@ -77,7 +66,6 @@ describe('use-column-widths', () => {
       });
 
       it('should return one size for hug and equal sizes for two columns with fill', () => {
-        measureTextMock.mockReturnValue({ width: 10 });
```

Comment:
Removing this did not actually change the outcome of the test so not sure that the intention was here.

Reply:
ahh, yes I noticed this to today. The width is smaller than the accepted min value, so that is why it returns 120. I have changed this in my PR